src/components/Input/Input.tsx

```diff
@@ -1,129 +1,92 @@
 import * as React from "react";
 import bem from "../../utils/bem";
-import TextField, { TextFieldOptions } from "../TextField/TextField";
-import Label, { LabelOptions } from "../Label/Label";
-import HelperErrorText, {
-    HelperErrorTextOptions,
-} from "../HelperErrorText/HelperErrorText";
-
-export interface InputOptions {
-    baseClass?: string;
-    blockName?: string;
+import { InputTypes } from "./InputTypes";
 
-    labelId?: string;
-    labelContent?: JSX.Element;
+export interface InputProps {
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
 
-    inputId?: string;
-    inputType?: string;
-    inputValue?: string | number;
-    onInputChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+    /** Populates the aria-label on the select */
+    ariaLabel?: string;
 
-    isRequired?: boolean;
+    /** Populates the aria-labelledby on the select */
+    ariaLabelledBy?: string;
 
-    helperContentOpts?: HelperErrorTextOptions;
-    errorContentOpts?: HelperErrorTextOptions;
+    /** Will add 'aria-required: true' to input */
```

Comment:
Should it also add a flag/label?

Reply:
Label's a separate component now, so the story shows them being used together (which is how we mostly want consumers to build forms, and what most of the DS designs call for).